### DISCUSSION TITLE
WDIO 7.0 - Mutiple HTML cucumber report - Aggregated report

### DISCUSSION THREAD

**Discussion by gopilaxman** [webdriverio/webdriverio#9799] [Q&A]
Mutiple HTML cucumber report in my repo. When I try to rerun the failed test case, It always adds new entry in the HTML report which shows duplicate entries in the report. And test case counts are increased including failed testcases + Passed Testcases. 
It makes little messy after couple of runs. is there a way to updated existing rows instead of adding new entries for every run ?
Is there a way to aggregate the report like the behavior in allure? Thanks in advance..

**Comment by christian-bromann** [comment]
You might want to ask this in the reporter repository.
